src/ert/config/queue_config.py

```diff
@@ -67,6 +69,12 @@ def from_dict(cls, config_dict: ConfigDict) -> QueueConfig:
                     " usually provided by the site-configuration file, beware that"
                     " you are effectively replacing the default value provided."
                 )
+            if (
+                values
```

Comment:
Shall we produce a warning that `setting SUBMIT_SLEEP for a different queue`?

Reply:
In this PR, you are only allowed to set it for the LSF queue, for which there is no driver for Scheduler. So that should not be necessary. 

Through this PR it submit_sleep is not really usable from an ert config. I think we can accept that until the LSF driver is merged, because then it should work immediately.

When job_queue is purged, we can add SUBMIT_SLEEP to GENERIC_QUEUE_OPTIONS  and it will work for all queues.

Reply:
(you will get a ConfigValidationError if you currently use SUBMIT_SLEEP with the local queue)